docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -139,6 +139,29 @@ The body of the request to /projects/$ProjectName will be an entity of the form:
 }
 ```
 
+```
+GET /projects/$ProjectName/images/
+```
+
+Will return a JSON object of the form
+
+```js
+{
+  "Images" : [
+    {
+      "candidate": "123456",
+      "visit": "V1",
+      "scan_type": Acquisition protocol ex :"t2",
+      "link": URL relative to this API ex: "\/candidates\/300022\/V1\/images\/loris-MRI_123456_V1_t2_001.mnc",
+      "insert_time": The inserted date ISO 8601  ex: "2016-08-09T14:15:30-04:00"
+    },
+    ...
+  ]
+}
+```
+It is possible to provide a parameter named `since` where the value need to be a date or datetime ex: 2016-08-09 or 2016-08-09 10:00:00 or 2016-08-09T10:00:00
```

Comment:
This should specify a GET parameter to avoid ambiguity of whether POST parameters are accepted.

The example values need quotation marks, and should probably only have one option to simplify things and avoid ambiguity. It'd probably be best to take an existing standardized date format that's close enough to the last option ( RFC3339? There's a built in constant in PHP for it, too, which should make it easy to parse: http://php.net/manual/en/class.datetime.php#datetime.constants.rfc3339) and reference that.